README.md

```diff
@@ -2,41 +2,26 @@
 
```

Comment:
Could be a good opportunity to add a badge?

`[![continuous-integration](https://github.com/executablebooks/github-activity/actions/workflows/tests.yaml/badge.svg)](https://github.com/executablebooks/github-activity/actions/workflows/tests.yaml)`